file-source/README.adoc

```diff
@@ -0,0 +1,33 @@
+= Spring Cloud Stream File Sink
```

Comment:
Found it; also the parent pom change etc; I did a `git add .` from within the module.